sentry/src/main/java/io/sentry/Hub.java

```diff
@@ -154,7 +156,9 @@ public SentryId captureEnvelope(
     } else {
       try {
         final StackItem item = stack.peek();
-        sentryId = item.getClient().captureException(throwable, item.getScope(), hint);
+        final SentryEvent event = new SentryEvent(throwable);
```

Comment:
THis LGTM, we could just bring back the default methods on `ISentryClient` and merge